### PR TITLE
A: (nsfw) http://thumbnails.porncore.net/lovespinkmatter/624722414502…

### PR DESCRIPTION
**Pull request by Yuki2718**
…363136/ (fixes https://github.com/AdguardTeam/AdguardFilters/issues/70558)
https://github.com/AdguardTeam/AdguardFilters/issues/70558

### CODE REVIEW

File: easyprivacy/easyprivacy_general.txt [hunk 1]
```diff
@@ -3046,6 +3046,8 @@
 /no-impression.gif?
 /nofastat.
 /notifications/analytics
+/noui/eventlogger.js
+/noui/StatProvider.js
```
Comment (spirillen) [inline easyprivacy/easyprivacy_general.txt:3050]: I don't see these today...

<details><summary>Logger output</summary>

| | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +226 | \|\|googletagmanager.com^ | -- | thumbnails.porncore.net | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-3598761-4` |
| +226 | \|\|porncore.net/*.php | -- | thumbnails.porncore.net | 1 | script | `http://thumbnails.porncore.net/jgkmraadutuszx.php` |
| +52 | \|\|googletagmanager.com^ | -- | thumbnails.porncore.net | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-3598761-4` |
| +52 | \|\|porncore.net/*.php | -- | thumbnails.porncore.net | 1 | script | `http://thumbnails.porncore.net/jgkmraadutuszx.php` |
| +0 | \|\|googletagmanager.com^ | -- | thumbnails.porncore.net | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-3598761-4` |
| +0 | \|\|porncore.net/*.php | -- | thumbnails.porncore.net | 1 | script | `http://thumbnails.porncore.net/jgkmraadutuszx.php` |
</details>

Comment (Yuki2718) [reply easyprivacy/easyprivacy_general.txt:3050]: <details>

porncore

</details>

Comment (spirillen) [reply easyprivacy/easyprivacy_general.txt:3079]: Cool, you even got another domain (`pc20160301.com`), witch I didn't see at all :smile: 

Chose I saw was 

```
31.media.tumblr.com
40.media.tumblr.com
64.media.tumblr.com
69.media.tumblr.com
assets.tumblr.com
counter.yadro.ru
pc180101.com
pinduck.com
thebestfreeporn.net
www.googletagmanager.com
www.redditstatic.com
```

Thanks for your follow up :+1:

**Comment by Yuki2718** [comment]
All the rules are still working and needed.

**Pull request closed without merging**